src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -19,21 +20,31 @@ public static final class Drivetrain {
 	}
 	
 	public static final class Arm {
-		
+		// Array Indexes (Just to make things easier to read)
+        public static final int ARM = 0;
+		//Motor port
+		public static final int MOTOR_PORT1 = 7;
+		public final static int MOTOR_PORT2 = 8;
 		//all angles in rot here
 		public static final double intakeAngle = -.1;
 		public static final double ampAngle = .3;
 		public static final double speakerAngle = .4;
 		//if needed
-//		public static final trapAngle = 80;
-		
+		public static final double UPPER_ANGLE = Math.toRadians(70); 
+		public static final double LOWER_ANGLE = Math.toRadians(0);
+		public static final double ARM_DICONT_RAD = (LOWER_ANGLE + UPPER_ANGLE) /2 - Math.PI;
 		//Motor Controllers: pid, FF
 		public static final double[] pidVals = new double[] { 0.1, 0.0, 0.1 };
 		// Feedforward
 		public static final double kS = 0.1;
 		public static final double kG = 0.1;
 		public static final double kV = 0.1;
 		public static final double kA = 0.1;
```

Comment:
For consistency if you make pid values into an array, also make feedforward values into an array. You can also make separate double variables too, up to you.

Reply:
-1
Leave feedforward values out of array, but simply append ff_ to them.
Feedforward values are not self-explanatory by order, unlike P-I-D.